browser/src/control/Control.StatusBar.js

```diff
@@ -585,6 +591,11 @@ L.Control.StatusBar = L.Control.extend({
 
 		toolbaritems.push({ id: 'morelanguages', text: _('More...') });
```

Comment:
I'm wondering if `More...` should be then replaced with: `Set language for All` to be unified with other entries

Reply:
cc @pedropintosilva

Reply:
to me, it makes "more..." sense 😂 . agreed to @eszkadev

Reply:
Sorry, I missed this notification.

Thanks @lpranam yup let's please replace `More...` with `Set Language for All text`